### PR TITLE
Add wagtail 3 support

### PR DESCRIPTION
**Pull request by fabienheureux**
Since wagtail 3 is out, imports from wagtail.core have changed 
This PR brings wagtail 3.0 support

I am using this branch on a real wagtail installation without issue so far.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
     url='https://github.com/neon-jungle/wagtail-metadata',
 
     install_requires=[
-        'wagtail~=2.0',
+        'wagtail~=3.0',
```
Comment (seb-b) [inline setup.py:21]: If we set the minimum version to 3, we won't need the try/except around the imports. I'm ok with releasing a major version for 3.0 support to no have the old imports hanging around

Comment (fabienheureux) [reply setup.py:21]: Ah...good catch :) 

Is it that bad to keep old imports ? Wagtail 3.0 was just released and not even LTS.
We could support wagtail 2 until the next LTS is out, and add a warning in changelog about future deprecation ?

Comment (seb-b) [reply setup.py:21]: it's not bad, it's just not super necessary. The current version of this plugin works with any wagtail 2.x, so happy to cut a new release for the next version, a note in the readme wouldn't hurt either

Comment (fabienheureux) [reply setup.py:21]: Fair enough, I am happy for you to just release a new version then. 
I will update my PR

**Pull request merged**